Sources/MinderaPeople-iOS-DesignSystem/Components/Buttons/ButtonPallet.swift

```diff
@@ -0,0 +1,18 @@
+import SwiftUI
+
+struct ButtonPallet {
+    let lightMode: ButtonPalletColorScheme
+    let darkMode: ButtonPalletColorScheme
+}
+
+struct ButtonPalletColorScheme {
```

Comment:
these are all internal types and nesting would only bring boilerplate button style usage. Is there any advantage on nesting that I'm not seeing in this case?

Reply:
Being internal and relevant/relative to another type (`ButtonPallet`) is precisely why I think nesting makes sense. I don't see how it will increase boilerplate, but if you think it does then please disregard my comment.

Reply:
I reach to a solution respecting that and reducing the extra code I think